toolkit/tools/internal/safemount/main_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package safemount
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+)
+
+var (
+	tmpDir     string
+	workingDir string
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I'm not super familiar with `main_test.go`, we have generally dropped `TestMain()` into the same `<pkg>_test.go` files as the other tests. Is there a benefit to splitting this out into its own file?

Reply:
The `TestMain()` function is for the entire package. So, it makes sense to keep it in a separate file. Granted, it doesn't matter much in this package since there is only a single file. But this is the way I've done it elsewhere. So, I am copying it here.